api-spec/aura-framefx-api.yaml

```
@@ -0,0 +1,628 @@
+        - taskId
+        - status
+
+  # Common Responses
+  responses:
+    BadRequestError:
+      description: Invalid request format or parameters
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)